Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN npm install --silent
 COPY frontend/ /code/
 RUN npm run build
 
-FROM python:3.8-slim as base
+FROM python:3.8-slim-bullseye as base
```

Comment:
Did we confirm that adding "bullseye" here was actually necessary?  Fine with leaving it in to be explicit, but I'm not sure if it will lead to questions later.

Reply:
I think it's better to be explicit here, so the base distro doesn't change out from underneath us. So technically it isn't necessary, but I think it'll help later to prevent deploys suddenly failing when Debian `bookworm` is released down the road and the Python base image switches to it.